requirements.txt

```diff
@@ -1,2 +1,3 @@
-databricks-sql-connector>=2.0.2
+databricks-sql-connector>=2.0.2; python_version<'3.10'
+databricks-sql-connector>=2.0.2,<2.0.3; python_version=='3.10'
```

Comment:
Why do we need this line?

Reply:
It applies the upper bound only when Python 3.10.